### DISCUSSION TITLE
Text inputs are not editable once converted to PDF

### DISCUSSION THREAD

**Discussion by WKampel** [gotenberg/gotenberg#977] [Q&A]
I'm converting an html form document to pdf.

The inputs and textareas are not modifiable after the conversion.

Does Gotenberg support inputs?

**Comment by gulien** [comment]
Hello @WKampel 👋 

Gotenberg is a glorified wrapper around other tools. So if the underlying tool does not support inputs, there is not much I can do.

I guess you’re using the Chromium module, maybe you could try to convert your HTML with the LibreOffice module; it may translate to a better result.

**Comment by WKampel** [comment]
I just tried with LibreOffice. The output looks strange, and the inputs/textareas do not even show up for some reason.

I know that many pdf forms have blue text inputs, surely there's a way to do it that I'm missing?

**Reply by gulien** [reply]
No idea alas 🤷‍♂️
